docs/rules/README.md

```diff
@@ -229,6 +228,7 @@ These rules are only relevant to ES6 environments.
 * [no-const-assign](no-const-assign.md) - disallow modifying variables that are declared using `const` (recommended)
 * [no-dupe-class-members](no-dupe-class-members.md) - disallow duplicate name in class members (recommended)
 * [no-new-symbol](no-new-symbol.md) - disallow use of the `new` operator with the `Symbol` object (recommended)
+* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified ES6 imports
```

Comment:
Can you propose a specific syntax for these? How about...

```md
* [no-restricted-modules](no-restricted-modules.md) - restrict usage of specified modules when loaded by `require` statements
* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified modules when loaded by `import` statements
```

I assume you do not wish to change the rule name, `no-restricted-imports`, as well.